.prettierrc.json

```diff
@@ -0,0 +1,8 @@
+{
+  "printWidth": 120,
+  "singleQuote": true,
+  "trailingComma": "es5",
+  "tabWidth": 2,
+  "useTabs": false,
+  "semi": true
+}
```

Comment:
`trailingComma` default is `"es5"`: https://prettier.io/docs/en/options.html#trailing-commas
`tabWidth` default is 2: https://prettier.io/docs/en/options.html#tab-width
`useTabs` default is `false`: https://prettier.io/docs/en/options.html#tabs
`semi` default is `true`: https://prettier.io/docs/en/options.html#semicolons

I don't think these defaults recently (or ever) changed so you can omit them, but you can also keep them if you think it shouldn't be implicit.